cmd/util/util.go

```diff
@@ -33,3 +46,125 @@ func Index[E comparable](s []E, v E) int {
 	}
 	return -1
 }
+
+func MustBootstrapDatastore(t testing.TB, engine string) (storagefixtures.DatastoreTestContainer, storage.OpenFGADatastore, string, error) {
+	container := storagefixtures.RunDatastoreTestContainer(t, engine)
+
+	uri := container.GetConnectionURI(true)
+
+	var ds storage.OpenFGADatastore
+	var err error
+
+	switch engine {
+	case "postgres":
+		ds, err = postgres.New(uri, sqlcommon.NewConfig())
+	case "mysql":
+		ds, err = mysql.New(uri, sqlcommon.NewConfig())
+	default:
+		return nil, nil, "", fmt.Errorf("'%s' is not a supported datastore engine", engine)
+	}
+	require.NoError(t, err)
+
+	return container, ds, uri, nil
+}
+
+func PrepareTempConfigDir(t *testing.T) string {
+	_, err := os.Stat("/etc/openfga/config.yaml")
+	require.ErrorIs(t, err, os.ErrNotExist, "Config file at /etc/openfga/config.yaml would disturb test result.")
+
+	homedir := t.TempDir()
+	t.Setenv("HOME", homedir)
+
+	confdir := filepath.Join(homedir, ".openfga")
+	require.Nil(t, os.Mkdir(confdir, 0750))
+
+	return confdir
+}
+
+func PrepareTempConfigFile(t *testing.T, config string) {
+	confdir := PrepareTempConfigDir(t)
+	confFile, err := os.Create(filepath.Join(confdir, "config.yaml"))
+	require.Nil(t, err)
+	_, err = confFile.WriteString(config)
+	require.Nil(t, err)
+	require.Nil(t, confFile.Close())
+}
+
+type ValidationResult struct {
+	StoreID       string
```

Comment:
For consistency with our API, please add struct tags so that these JSON marshal consistently with our API protobufs.

`store_id`
`authorization_model_id`

We also tend to use snake case instead of camelCase, so `is_latest_model` would be preferred over `IsLatestModel`.